integration/setup/setup.go

```diff
@@ -77,8 +77,9 @@ func installAWSOperator(ctx context.Context, config Config) error {
 		c := chartvalues.AWSOperatorConfig{
 			Provider: chartvalues.AWSOperatorConfigProvider{
 				AWS: chartvalues.AWSOperatorConfigProviderAWS{
-					Encrypter: "kms",
-					Region:    env.AWSRegion(),
+					Encrypter:       "kms",
+					Region:          env.AWSRegion(),
+					RouteTableNames: env.AWSRouteTable0() + "," + env.AWSRouteTable1(),
```

Comment:
That is the most important fix here. We need to use the route tables of the fake control plane vpc.